lasagne/nonlinearities.py

```diff
@@ -60,6 +60,87 @@ def tanh(x):
     return theano.tensor.tanh(x)
 
 
+# scaled tanh
+class ScaledTanH(object):
```

Comment:
This I think is a matter of convention, but I just looked around in different tools and literature and I think @ebenolson is probably right and `ScaledTanh` is the much more widely used option.